packs/csi_openstack_cinder/templates/csi_openstack_cinder.nomad.tpl

```diff
@@ -0,0 +1,84 @@
+job [[ template "job_name" . ]] {
+  [[ template "region" . ]]
+
+  datacenters = [[ .csi_openstack_cinder.datacenters | toPrettyJson ]]
+
+  type = system
+
+  group "nodes" {
+
+    restart {
+      attempts = [[ .csi_openstack_cinder.job_restart_config.attempts ]]
+      delay    = [[ .csi_openstack_cinder.job_restart_config.delay ]]
+      mode     = [[ .csi_openstack_cinder.job_restart_config.mode ]]
+      interval = [[ .csi_openstack_cinder.job_restart_config.interval ]]
+    }
+    
+    [[ template "constraints" .csi_openstack_cinder.constraints ]]
+
+    [[- template "vault_config" .csi_openstack_cinder -]]
+
+    task "cinder-node" {
+      driver = "docker"
+      template {
+        data        = <<EOT
+[[ $config := .csi_openstack_cinder.cloud_conf_file ]][[ fileContents $config ]]
```

Comment:
I could use some clarification on the handling of the `cloud_conf_file` value.  I checked out this branch and ran it and got the following error:

```
$ nomad-pack run .
! Failed To Process Pack

        Error:   failed to render csi_openstack_cinder/templates/csi_openstack_cinder.nomad.tpl: template:
csi_openstack_cinder/templates/csi_openstack_cinder.nomad.tpl:25:57: executing
"csi_openstack_cinder/templates/csi_openstack_cinder.nomad.tpl" at <fileContents $config>:
error calling fileContents: failed to read : open : no such file or directory
        Type:    *errors.errorString
        Context:
                 - Registry Name: dev
                 - Pack Name: csi_openstack_cinder
                 - Pack Ref: dev
                 - Deployment Name: csi_openstack_cinder
```

So it looks like this is required (let's document that!)? So then I copied the `cloud.conf.example` file to cloud.conf` and that got the error I described above for argument quoting.

Reply:
Made the variable for providing the file path of the cloud config file a required var so this shouldn't be an issue going forward.